wrappers/javascript/nodejs/README.md

```diff
@@ -0,0 +1,26 @@
+# anoncreds-nodejs
```

Comment:
Will you do the renaming of the folders (to anoncreds-nodejs etc..) in a follow up PR?

We should make sure that change is also made in the askar repo